src/components/ADempiere/ContextMenu/contextMenuDesktop.vue

```diff
@@ -35,6 +35,9 @@
               {{ $t('components.contextMenuDownload') }}
             </a>
           </el-menu-item>
+          <el-menu-item v-show="$route.name === 'Report Viewer'" index="9" @click="$router.push({ name: '8e5131dc-fb40-11e8-a479-7a0060f0aa01' })">
+            {{ $t('components.contextMenuPrintFormatSetup') }}
```

Comment:
Please add a Util class for launch static components from UI and add criteria for open current 
```
@AD_PrintFormat_ID@
```

Reply:
Updated in this [commit](https://github.com/adempiere/adempiere-vue/pull/163/commits/c997fa27a4e0ef37759b8b0dd7c6ac3cf08fa939)